common-lib/src/cmr/common/api/web_server.clj

```diff
@@ -46,44 +47,92 @@
   report in Cubby can be in the 5 to 10 MB range."
  (* 50 ONE_MB))
 
-(defn- routes-fn-verify-size
+(def buffer-size
```

Comment:
I am wondering how big a difference different buffer sizes will make? and is it worthy to considering using different buffer sizes (vs one buffer size here) to help with the performance?